src/components/SpiceList.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react'
-// TODO: import the SpiceItem component here
+import SpiceItem from './SpiceItem';
 
 // class component
 class SpiceList extends React.Component {
 
-  renderSpices() {
-    // TODO: update this function to return an array of SpiceItem components
-    // If you get an error message about a 'key' prop, make sure to fix it before submitting your code!
+  renderSpices = (spices) => {
+    return (
+      spices.map(spice => <SpiceItem spice={spice} />)
```

Comment:
Make sure to include the `key` prop when you are returning an array of JSX, like this:

```jsx
spices.map(spice => <SpiceItem key={spice.id} spice={spice} />)
```

Also, you can access `this.props.spices` directly in your `renderSpices` function, so it's not necessary to pass them in as an argument:


```jsx
renderSpices() {
    return this.props.spices.map(spice => <SpiceItem key={spice.id} spice={spice} />)
}
```

Reply:
Yes, thanks